.storybook/main.js

```diff
@@ -0,0 +1,32 @@
+// your app's webpack.config.js
+const custom = require("../webpack.config.js")();
+
+module.exports = {
+  "stories": [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)"
```

Comment:
1. `mdx`?
2. Do you ever intend to write a story that's not `tsx`?

Reply:
I don't but when @alfetopito comes back he might

Reply:
In general, I don't want to remove what little storybook added on `init` by default. As it is a good example of what works, in case we break something later.